runtime/server/x86/CMakeLists.txt

```diff
@@ -39,8 +39,8 @@ include_directories(${glog_SOURCE_DIR}/src ${glog_BINARY_DIR})
 
 # third_party: gtest
 FetchContent_Declare(googletest
-  URL      https://github.com/google/googletest/archive/release-1.10.0.zip
-  URL_HASH SHA256=94c634d499558a76fa649edb13721dce6e98fb1e7018dfaeba3cd7a083945e91
+  URL      https://github.com/google/googletest/archive/release-1.11.0.zip
+  URL_HASH SHA256=353571c2440176ded91c2de6d6cd88ddd41401d14692ec1f99e35d013feda55a
```

Comment:
Why change the gflags and gtest version?

Reply:
1. the latest version of gflags: https://github.com/gflags/gflags/issues/267
2. the latest version of gtest: https://github.com/google/googletest/pull/2498